### PR TITLE
Open API Docs for /application endpoints

### PR DESCRIPTION
**Pull request by warunicorn19**
Signed-off-by: Aditya Chaterjee <speak2adi@gmail.com>

**Description**
Added swagger docs to Application endpoint. 
This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
Heads-up: a conflict - https://github.com/meshery/meshery/pull/3658/conflicts

**Comment by warunicorn19** [comment]
> Heads-up: a conflict - https://github.com/meshery/meshery/pull/3658/conflicts

Oh no.

**Comment by leecalcote** [comment]
This has the same PR subject as #3643. Is this a duplicate? // @sayantan1413

**Comment by warunicorn19** [comment]
@leecalcote no, actually @sayantan1413 and I are working on this together so these are different than the ones he's done.

**Comment by sayantan1413** [comment]
Yes @leecalcote this is for a different endpoint.😃

File: handlers/meshery_application_handler.go [hunk 1]
```diff
@@ -30,6 +30,20 @@ func (h *Handler) ApplicationFileHandler(
...
+// Sends requests for Application files
+// responses:
+// 	200: applicationFileRequestWrapper
+
+// swagger:route GET /api/experimental/application/ ApplicationsAPI idGetApplicationFileRequest
+// Handle GET request for Application Files
+
+// Fetches requests for Application files
+// responses:
+// 	200: noContentWrapper
```
Comment (piyushsingariya) [inline handlers/meshery_application_handler.go:45]: you sure? with GET we are getting no response?

File: handlers/meshery_application_handler.go [hunk 2]
```diff
@@ -30,6 +30,20 @@ func (h *Handler) ApplicationFileHandler(
 	h.PatternFileHandler(rw, r, prefObj, user, provider)
 }
 
+// swagger:route POST /api/experimental/application/ ApplicationsAPI idPostApplicationFileRequest
+// Handle POST request for Application Files
+
+// Sends requests for Application files
```
Comment (piyushsingariya) [inline handlers/meshery_application_handler.go:36]: I think we can have a better description here.

File: handlers/meshery_application_handler.go [hunk 3]
```diff
@@ -30,6 +30,20 @@ func (h *Handler) ApplicationFileHandler(
...
+// responses:
+// 	200: applicationFileRequestWrapper
+
+// swagger:route GET /api/experimental/application/ ApplicationsAPI idGetApplicationFileRequest
+// Handle GET request for Application Files
+
+// Fetches requests for Application files
```
Comment (piyushsingariya) [inline handlers/meshery_application_handler.go:43]: better description

File: handlers/doc.go [hunk 4]
```diff
@@ -242,3 +242,18 @@ type singleScheduleResponseWrapper struct {
 	// in: body
 	Body models.Schedule
 }
+
+// Fetches Meshery Application
```
Comment (piyushsingariya) [inline handlers/doc.go:246]:
```suggestion
// Fetches a Single Meshery Application
```

**Review by piyushsingariya — CHANGES_REQUESTED** [review]

File: handlers/meshery_application_handler.go (hunk 1 above)
Comment (warunicorn19) [reply handlers/meshery_application_handler.go:45]: I'm not a 100% sure, could you tell me if it is getting any response?

Comment (piyushsingariya) [reply handlers/meshery_application_handler.go:45]: Go through the handler maybe. That's where you'll find what's the response from the api.

**Comment by piyushsingariya** [comment]
Hey @warunicorn19 can we be a bit quick here??

**Comment by warunicorn19** [comment]
Yes sorry, I'll put in the PR tonight.

**Comment by warunicorn19** [comment]
hey @piyushsingariya, I've tried a lot to understand the handler, but couldn't understand it, i still think it is not returning anything. I'm really stuck😬.

**Comment by leecalcote** [comment]
Heads-up: a merge conflict.

**Comment by piyushsingariya** [comment]
> hey @piyushsingariya, I've tried a lot to understand the handler, but couldn't understand it, i still think it is not returning anything. I'm really stuckgrimacing.

That handler is a GET handler for applications, so of course, this may be returning some application struct. @sayantan1413 would you help @warunicorn19 here.

**Comment by sayantan1413** [comment]
Hey @warunicorn19 in the doc.go file add this
<pre> 
// Returns all meshery applications
// swagger:response mesheryApplicationsResponseWrapper
type mesheryApplicationsResponseWrapper struct {
	// in: body
	Body models.ApplicationsAPIResponse
}
</pre>

and use the `mesheryApplicationsResponseWrapper` in the GET response.

Also please resolve the merge conflict .

// @piyushsingariya  I think this is the required solution for the GET response for the Applications API .

**Comment by warunicorn19** [comment]
cool @sayantan1413 I'm closing this PR and creating a new one, because i sort of messed up on this branch

**Pull request closed without merging**